src/index.js

```diff
@@ -0,0 +1,172 @@
+const getWeatherBtn=document.getElementById("get-weather-btn");
+const skyDropDown=document.getElementById('sky-dropdown');
+const Sunny=document.getElementById('sunny');
+const Snowy=document.getElementById('snowy');
+const Rainy=document.getElementById('rainy');
+const Cloudy=document.getElementById('cloudy');
+const defaultSky=document.getElementById('default-sky');
+
+// ---------------------------
+
+let temp=0;
+colors();
+getApi();
+
+function getApi() {
+    let latitude;
+    let longitude;
+        axios
+        .get(`http://127.0.0.1:5000/location?q=${userSearch.value}`)
+        .then(function apiResponse(response) {
+            latitude = response.data[0].lat;
+            longitude = response.data[0].lon;
+
+            axios
+            .get(`http://127.0.0.1:5000/weather?&lat=${latitude}&lon=${longitude}`)
+            .then((res) => {
+                const fahrenheitConversion = Math.floor(
+                ((res.data.main.temp - 273.15) * 9) / 5 + 32);
+                temperature.textContent = fahrenheitConversion;
+                temp = fahrenheitConversion;
+                seasonChange()
+                })
+            })
+            .catch((error) => {
+                alert(error);
+            })    
+};
+
+temperature.textContent = temp
+
+function increment(){
+    temp += 1;
+    temperature.textContent=temp
+    console.log(temp)
+    colors()
+    seasonChange()
+};
+
+function decrement(){
+    temp -= 1;
+    temperature.textContent=temp
+    console.log(temp)
+    colors()
+    seasonChange()
+};
+
+function colors(){
+    if (temp >= 90){
+        temperature.style.color="#ab0000"
+    }else if (temp <= 89 && temp >= 80){
+        temperature.style.color="red"
+    }else if (temp <= 79 && temp >= 70){
+        temperature.style.color="orange"
+    }else if (temp <= 69 && temp >= 60){
+        temperature.style.color="green"
+    }else if (temp <= 59 && temp >= 50){  
+        temperature.style.color="#38f9cc"
+    }else if (temp <= 49 && temp >= 40){  
+        temperature.style.color="#378aea"
+    }else if (temp <= 39 && temp >= 30){  
+        temperature.style.color="blue"        
+    }else {
+        temperature.style.color="#1100ab"
+    }           
+};
+
+function seasonChange(){
+    if (temp >= 80){
+        summer.style.display="flex"
+        spring.style.display="none"
+        fall.style.display="none"
+        winter.style.display="none"
+    }else if (temp >= 65){
+        summer.style.display="none"
+        spring.style.display="flex"
+        fall.style.display="none"
+        winter.style.display="none"
+    }else if (temp >= 45){
+        summer.style.display="none"
+        spring.style.display="none"
+        fall.style.display="flex"
+        winter.style.display="none"
+    }else if (temp <= 44){
+        summer.style.display="none"
+        spring.style.display="none"
+        fall.style.display="none"
+        winter.style.display="flex"
+    }
```

Comment:
So here, the refactor that I would suggest would start in the HTML file. Instead of putting all the landscapes I would just put one of them. Something like : `<div id='landscape'></div>` After that, I would change the code you have here into something like: 

```javascript
const landscape = document.getElementById('landscape')

if (temp >= 65) {
   landscape.innerHTML = 😎S☀️u🌞n🌅n🌞y😎
}
```

This way you can save your fingers from typing a lot more code! I know you have a top and bottom landscape so you would just change them bottom by their respective ids!